docs/source/app_developers_guide/go_sdk_import.rst

```diff
@@ -0,0 +1,59 @@
+********************
+Importing the Go SDK
+********************
+
+.. note::
+   The Sawtooth Go SDK assumes that you have the latest version of Go,
+   more information on installing and configuring can be found at `go
+   <https://github.com/golang/go>`_.
+
+Once you've got a working version of Sawtooth, there are a few additional
+steps you'll need to take to get started developing for Sawtooth in Go.
+
+1. Get Go SDK located at `Sawtooth Go SDK repository
+   <https://github.com/hyperledger/sawtooth-sdk-go>`_.
+
+.. code-block:: console
+    :caption: Run following command for downloading Go SDK
```

Comment:
The rest of the Sawtooth documentation does not use captions for command examples. I suggest removing all captions in this file, because they duplicate the information in the preceding sentence. (If you keep the captions, please make them shorter and correct the grammar.)